src/sql/src/pure.rs

```diff
@@ -382,13 +363,22 @@ async fn purify_csr_connector_proto(
     };
 
     let CsrConnectorProto {
-        url,
+        connector,
         seed,
         with_options: ccsr_options,
     } = csr_connector;
     match seed {
         None => {
-            let url: Url = url.parse()?;
+            let url: Url = match connector {
+                CsrConnector::Inline { url: uri } => uri.parse()?,
+                CsrConnector::Reference { url, .. } => match url {
+                    Some(url) => {
+                        info!("parsing {} into url", url);
```

Comment:
This kind of thing is great for debugging, but to be useful in system logging, you'd likely want to include more context for some random DBA/sales engineer as to why this could point to a problem in the first place--i.e. put yourself in the user's shoes

Reply:
Yup, this should've been removed. The big lesson for me here is this PR got too big for me to keep track of all of these type of detail and clean it up properly before asking for review. ;)